src/deploy/schedule.rs

```diff
@@ -37,12 +37,14 @@ impl ScheduleTarget {
             .body(build_schedules_request(&self.crons))
             .send()?;
 
-        if !res.status().is_success() {
-            anyhow::bail!(
-                "Something went wrong! Status: {}, Details {}",
-                res.status(),
-                res.text()?
-            )
+        let status = res.status();
+        if !status.is_success() {
+            let text = res.text()?;
+            if text.contains("workers.api.error.not_entitled") {
```

Comment:
Is durable objects the only possible reason for not_entitled?

Reply:
Probably not, but it's a temp quick fix until you get DO's to GA.

Reply:
Then is there a reason not to show the original error plus a "If you are trying to use Modules, you need to opt in..."?

As it stands someone's probably going to be more confused, not less, if it has nothing to do with modules or DO

Reply:
I'd think there would be a better API error, actually indicating the accurate entitlement missing. But I'm assuming you're not ready to commit to that until GA..

The original error was "Something went wrong!" + the ambiguous `workers.api.error.not_entitled`.